### PR TITLE
Home Page Edits

### PR DESCRIPTION
**Pull request by SupaflyFPV**
I've finessed and tweaked the Home page text. Let me know what you think. I won't be offended if you are unhappy with it. My goal was to make it more simple and succinct, and portray Betaflight in the way I see it.

### CODE REVIEW

File: src/components/Home/index.tsx [hunk 1]
```diff
@@ -78,43 +82,29 @@ export default function Home({ recentPosts }: BlogProps) {
         <HomepageFeature title="About" compact={true}>
           <div className="grid grid-cols-1 md:grid-cols-2 xl:grid-cols-3 gap-4 xl:gap-6 w-full">
             <AboutCard title="Hardware" className="text-rose-500 fill-rose-500" Icon={CpuChipIcon}>
-              <p>Betaflight supports a wide range of flight controllers with at least an STM32F4 MCU</p>
-              <p>You can use nearly any piece of modern hardware out there. Full telemetry and high refresh rate for receivers, camera and VTX control, RGB LEDs, and much more!</p>
-              <p>Run the configurator on any device, whether it is using Windows, MacOS, Linux, or Android</p>
+              <p>Betaflight supports a wide range of flight controllers from a variety of manufacturers.</p>
+              <p>The 'Approved Partner' program provides hardware recommendations from the development team based.</p>
```
Comment (VitroidFPV) [inline src/components/Home/index.tsx:86]: Based? Typo or intended?

File: src/components/Home/index.tsx [hunk 2]
```diff
@@ -78,43 +82,29 @@ export default function Home({ recentPosts }: BlogProps) {
...
             <AboutCard className="border-purple-600 text-purple-600" title="Flight Dynamics" Icon={JetIcon}>
-              <p>
-                Betaflight not only strives to put the best flight code on your controller, it also looks to satisfy such requirements as perfect performance, tiny footprint, full feature set and a
-                strong quality assurance process.
-              </p>
-              <p>
-                To assist in this endeavour all flight related data can be logged into a blackbox for later analysis. So pilots and technicians can objectively base their tune on the most appropriate
-                data.
-              </p>
+              <p>Betaflight was created for cutting edge flight performance. This has been achieved optimising reaction time to disturbances, and increasing the accuracy of stick tracking.</p>
```
Comment (VitroidFPV) [inline src/components/Home/index.tsx:107]: [...] been achieved \*by* optimi\*z*Ing [...]

**Review by VitroidFPV — COMMENTED** [review]
A few typos/grammar mistakes

File: src/components/Home/index.tsx (hunk 1 above)
Comment (SupaflyFPV) [reply src/components/Home/index.tsx:86]: the team is based

https://www.urbandictionary.com/define.php?term=based

Comment (SupaflyFPV) [reply src/components/Home/index.tsx:86]: ...yes it is a typo...must have left it in from an edit ;)

File: src/components/Home/index.tsx (hunk 2 above)
Comment (SupaflyFPV) [reply src/components/Home/index.tsx:107]: ...well I'm english and we invented the language ;)

Comment (VitroidFPV) [reply src/components/Home/index.tsx:107]: S/Z was just my keyboard correcting me, I'm honestly lost on the spelling there between UK/US, sorry 😅

**Review by sugaarK — APPROVED** [review]

File: src/components/Home/index.tsx (hunk 1 above)
Comment (SupaflyFPV) [reply src/components/Home/index.tsx]: Ok cleared that up...

File: src/components/Home/index.tsx (hunk 2 above)
Comment (SupaflyFPV) [reply src/components/Home/index.tsx:107]: No probs, it can be confusion. I think considering our UK contingent we can let this version stand ;)

**Comment by nerdCopter** [comment]
for anyone wanting to compare:
image

**Comment by SupaflyFPV** [comment]
> for anyone wanting to compare: image

yeah in this section I wanted to cut out specific info as that would be better places elsewhere IMO. Tiles should be used as a topline IMO. If anyone thinks that certain bits need to be there or preferable let me know.

**Comment by haslinghuis** [comment]
Maybe have a QA section

image

**Review by sugaarK — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**